docs/src/components/CodeBlock.tsx

```diff
@@ -71,7 +72,9 @@ export const CodeBlock: React.FC<CodeBlockProps> = ({
                     <LivePreview />
                   </LiveProvider>
                 </div>
-                <ShowHideButton hide={hide} onHideChange={setHide} />
+                <ThemeProvider theme={b2bTheme}>
```

Comment:
Shouldn't the switch theme stuff be here?

Reply:
Then you have the problem that you have to change the theme for every component example separately. Copy/show button should have the initial theme

Reply:
Not quite sure I follow. You can add the switch globally and then pass it to the `<ThemeProvider>` inside the `<Codeblock!>` :)